types/src/client.rs

```diff
@@ -17,6 +17,18 @@ pub struct Subscription<Notif> {
 	pub marker: PhantomData<Notif>,
 }
 
+/// Active NotificationHandler on a Client.
+pub struct NotificationHandler<Notif> {
+	/// Channel to send requests to the background task.
+	pub to_back: mpsc::Sender<FrontToBack>,
+	/// Channel from which we receive notifications from the server, as encoded `JsonValue`s.
+	pub notifs_rx: mpsc::Receiver<JsonValue>,
+	/// Method,
```

Comment:
```suggestion
	/// Method name
```
Maybe it'll be clearer once I read through, but why is this needed?

Reply:
It's need because the user declares that wants to "subscribe" for all notifications for `method` then this is used to filter out notifications that the user it interested in.

It's in essence similar to `subscriptions` but then the subscription ID is used for that instead.